includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -320,6 +320,7 @@ private function getLoggedInUserData( $user ) {
 
 		return [
 			'header' => [
+				'id' => strval( $user->getId() ),
```

Comment:
why do we need it? + I'm not sure if we're still using both models for Navigation

Reply:
We need it to check if the target user of a notification is the user that is reading this notification. If so we display another kind of message.